src/pages/projectSettings/ProjectSettingsTabs.tsx

```diff
@@ -0,0 +1,116 @@
+import { useEffect } from "react";
+import styled from "@emotion/styled";
+import { H2, Disclaimer } from "@leafygreen-ui/typography";
+import { Route, useParams } from "react-router-dom";
+import { routes, ProjectSettingsTabRoutes } from "constants/routes";
+import {
+  AccessTab,
+  EventLogTab,
+  GeneralTab,
+  GitHubCommitQueueTab,
+  NotificationsTab,
+  PatchAliasesTab,
+  PeriodicBuildsTab,
+  ProjectTriggersTab,
+  VariablesTab,
+  VirtualWorkstationTab,
+} from "./projectSettingsTabs/index";
+
+export const ProjectSettingsTabs: React.FC = () => {
+  const { tab } = useParams<{ tab: string }>();
+
+  useEffect(() => {}, [tab]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  const { title, subtitle } = getTitle(tab as ProjectSettingsTabRoutes);
+
+  return (
+    <Container>
+      <TitleContainer>
+        <H2 data-cy="project-settings-tab-title">{title}</H2>
+        {subtitle && <Subtitle>{subtitle}</Subtitle>}
+      </TitleContainer>
+
+      <Route path={routes.projectSettingsGeneral} component={GeneralTab} />
+      <Route path={routes.projectSettingsAccess} component={AccessTab} />
+      <Route path={routes.projectSettingsVariables} component={VariablesTab} />
+      <Route
+        path={routes.projectSettingsGitHubCommitQueue}
+        component={GitHubCommitQueueTab}
+      />
+      <Route
+        path={routes.projectSettingsNotifications}
+        component={NotificationsTab}
+      />
+      <Route
+        path={routes.projectSettingsPatchAliases}
+        component={PatchAliasesTab}
+      />
+      <Route
+        path={routes.projectSettingsVirtualWorkstation}
+        component={VirtualWorkstationTab}
+      />
+      <Route
+        path={routes.projectSettingsProjectTriggers}
+        component={ProjectTriggersTab}
+      />
+      <Route
+        path={routes.projectSettingsPeriodicBuilds}
+        component={PeriodicBuildsTab}
+      />
+      <Route path={routes.projectSettingsEventLog} component={EventLogTab} />
+    </Container>
+  );
+};
+
+export const getTitle = (
```

Comment:
Do you think we might need this in other pages apart from the `ProjectSettings` Page itself and the `ProjectSettingsTabs` component above? If so it might make sense to move this to a more shared constants or utils file in this same directory

Reply:
Just had another look through the Figma and I don't really think so — but will keep this in mind if that does come up!